app/(public)/components/paragraph/stanford-card.tsx

```diff
@@ -15,8 +15,7 @@ interface CardProps {
 }
 
 const StanfordCard = ({paragraph, siblingCount, ...props}: CardProps) => {
-  const horizontal = paragraph?.behavior_settings?.sul_card_styles?.orientation;
-  const Card = horizontal ? HorizontalCard : VerticalCard;
+  const isHorizontal = paragraph?.behavior_settings?.sul_card_styles?.orientation === 'horizontal';
```

Comment:
Is this to avoid assigning this as `undefined`?

Reply:
yeah. it forces the variable to be a boolean which is much easier to have conditions on. because what if (random scenario) you have `paragraph?.behavior_settings?.sul_card_styles?.orientation` evaluate to `foobar`. It's still considered "horizontal" which isn't what we actually want.